ftp-edi-message-to-salesforce-opportunity/README.md

```diff
@@ -0,0 +1,82 @@
+# FTP B2B EDI message to Salesforce Opportunity
+
+This sample reads EDI files from a given FTP location, converts those EDI messages to Ballerina records, and creates a Salesforce Opportunity for each EDI message.
```

Comment:
Link to salesforce?

Reply:
Where to put the link?
Also to what we should link? salesforce homepage?